.misc/deploy.testpypi.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Both travis and rultor run deploy.pypi.sh, and actions depends on who's calling them?
Wouldn't that cause problems later?